Source/carthage/Extensions.swift

```diff
@@ -112,6 +112,9 @@ internal struct ProjectEventSink {
 
 		case let .skippedBuilding(project, message):
 			carthage.println(formatting.bullets + "Skipped building " + formatting.projectName(project.name) + " due to the error:\n" + message)
+
+		case let .skippedInstallingBinaries(project, message):
+			carthage.println(formatting.bullets + "Skipped installing " + formatting.projectName(project.name) + ".framework binary due to the error:\n\t" + formatting.quote(message))
```

Comment:
This is related to my comments on errors above. We can possibly remove the message. It seems like it would be disingenuous to say:

```bash
*** Skipped installing Result.framework binary built with Swift 2.2
```

when in fact, it was skipped because the parsing of the local swift version had failed. Either way, I do think it is very helpful to have the extra verbosity of local and framework swift versions here where possible. I think it's very useful when looking at CI logs--for instance--to figure out if it was the Xcode version change or a `carthage update` that is now causing your build to take much longer.